examples/HP34401A_measure_currents.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#    HP34401A_measure_currents.py - functions with PyVisa to monitor currents
+#      using a HP34401A multimeter. It supposes that there is only one GPIB
+#      instrument (the HP34401A multimeter) plugged to the computer.
+#      To use this example just put the example in the directory where you
+#      call python then just run:
+#      $> python
+#      >>> from HP34401A_measure_currents.py import *
+#      >>> contMeasA( "file.csv" )
+#      Hit Ctrl-C when you have finished
+#      Use your favorite plotting program to review the measurements (GnuPlot
+#       is a good candidate)
+#      $> gnuplot
+#      >>> set datafile separator ";"
+#      >>> plot 'file.csv' using 1:2 with lines
+#
+#    Copyright © 2014
+#                Manuel Yguel <manuel.yguel@strataggem.com>,
+#
+#    This file is part of PyVISA.
+#
+#    PyVISA is free software; you can redistribute it and/or modify it under
+#    the terms of the MIT licence:
+#
+#    Permission is hereby granted, free of charge, to any person obtaining a
+#    copy of this software and associated documentation files (the "Software"),
+#    to deal in the Software without restriction, including without limitation
+#    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#    and/or sell copies of the Software, and to permit persons to whom the
+#    Software is furnished to do so, subject to the following conditions:
+#
+#    The above copyright notice and this permission notice shall be included in
+#    all copies or substantial portions of the Software.
+#
+#    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#    DEALINGS IN THE SOFTWARE.
+#
+
+
+import visa, string
+
+
+# get ressources
+def initVisa():
+    return visa.ResourceManager()
```

Comment:
Why are you building a function for this? It is quite small piece of code. Also the resource manager should be instantiated once in your whole app.